include/deal.II/matrix_free/fe_evaluation.h

```diff
@@ -9538,6 +9555,52 @@ FEFaceEvaluation<dim,
 
 
 
+template <int dim,
+          int fe_degree,
+          int n_q_points_1d,
+          int n_components_,
+          typename Number,
+          typename VectorizedArrayType>
+template <typename StreamType>
+void
+FEEvaluation<dim,
+             fe_degree,
+             n_q_points_1d,
+             n_components_,
+             Number,
+             VectorizedArrayType>::print(StreamType &out) const
+{
+  out << " maximal precompiled degree: "
+      << internal::FEEvaluationFactory<dim, Number, VectorizedArrayType>::
+           max_precompiled_degree()
```

Comment:
This implementation really just prints a single piece of information. That doesn't match the documentation of the function, which suggests that it prints some kind of "summary" of things.